test/specs/register-unicon-breeder.e2e.js

```diff
@@ -0,0 +1,157 @@
+import { browser, expect } from '@wdio/globals'
+import { before } from 'mocha'
+import path from 'path'
+
+import nameEntryPage from '../page-objects/name.page.js'
+import emailPage from '../page-objects/email.page.js'
+import phoneNumberPage from '../page-objects/phone-number.page.js'
+import addressPage from '../page-objects/address.page.js'
+import certificateAddressPage from '../page-objects/certificate-address.page.js'
+import policyStartDatePage from '../page-objects/policy-start-date.page.js'
+import uploadFilePage from '../page-objects/upload-file.page.js'
+import selectNoOfUnicornsPage from '../page-objects/select-no-of-unicorns.page.js'
+import selectTypeOfUnicornsPage from '../page-objects/select-type-of-unicorns.page.js'
+import unicornsTextPage from '../page-objects/unicorns-text.page.js'
+import noOfUnicornsStaffPage from '../page-objects/no-of-unicorns-staff.page.js'
+import summaryPage from '../page-objects/summary.page.js'
+
+describe('Register unicorn breeder form - e2e', () => {
+  before(async () => {
+    await nameEntryPage.open()
+  })
+
+  it('should enter name', async () => {
+    await expect(browser).toHaveTitle(`What's your name? - e2e form - GOV.UK`)
+    await nameEntryPage.enterName.setValue('John Doe')
+    await nameEntryPage.submitButton.click()
+
+    await expect(browser).toHaveTitle(
+      `What's your email address? - e2e form - GOV.UK`
+    )
+  })
+
+  it('should enter email address', async () => {
+    await emailPage.enterEmail.setValue('john.doe@example.co.uk')
+    await emailPage.submitButton.click()
+    await expect(browser).toHaveTitle(
+      `What's your phone number? - e2e form - GOV.UK`
+    )
+  })
+
+  it('should enter phone number', async () => {
+    await phoneNumberPage.enterPhoneNumber.setValue('07779875')
+    await phoneNumberPage.submitButton.click()
+
+    await expect(browser).toHaveTitle(
+      `What's your address? - e2e form - GOV.UK`
+    )
+  })
+
+  it('should enter address', async () => {
+    await Promise.all([
+      addressPage.enterAddressLine1.setValue('1 High Street'),
+      addressPage.enterTown.setValue('Townsville'),
+      addressPage.enterPostcode.setValue('TS1 1ST')
+    ])
+    await addressPage.submitButton.click()
+
+    await expect(browser).toHaveTitle(
+      'Do you want your unicorn breeder certificate sent to this address? - e2e form - GOV.UK'
+    )
+  })
+
+  it('should select Yes to use same address for certificate', async () => {
+    await certificateAddressPage.selectYesOrNo.click()
```

Comment:
How do you feel about ditching all the page objects? 😱 

As we add more forms it's going to get very hard to manage

E.g. It's a lot simpler to add all the selectors inline:

```suggestion
    const $select = await $('aria/Yes')
    const $button = await $('aria/Continue')

    await $select.click()
    await $button.click()
```

Reply:
I thought its wdio best practise to separate them :) and e2e.js now has only form related things in it and page objects has selectors which seems nice distinction but I also agree to your point. 

Lets discuss

Reply:
My understanding is we will not add lot of forms to e2e tests. We will only have a user journey in e2e which will cover most scenarios

Reply:
I'd prefer to keep our existing test style with selectors inline

Or alternatively it's nice to set up Given, When, Then style:
https://github.com/webdriverio/cucumber-boilerplate/blob/main/src/features/checkbox.feature

See what everyone else thinks

Reply:
+1 for given-then-when format, that will be much easier to maintain in the long run.

When we need classes, perhaps it'd be better to use them for the component types rather than the pages as a whole? A page may have many components on it and question titles on those pages may differ between forms. It'd be more useful to have a more generic classes that just support the component inputs.

Reply:
I continued tests with existing format which is also how CDP do their tests. I believe our repo is cloned from cdp, so we took all config from them. We could use cucumber given/when/then format if we started tests that way but is it really necessary to change now and spend more time? I am not sure.